runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -128,6 +153,35 @@ private boolean isInKingdom()
 			&& KINGDOM_REGION.contains(client.getLocalPlayer().getWorldLocation().getRegionID());
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() != WidgetID.KINGDOM_GROUP_ID || !config.showKingdomValue())
```

Comment:
I see you've tried to make a guard clause, but you've just broken the code. See NightFireCat's comment for what the guard clause should look like